debian/control

```diff
@@ -0,0 +1,15 @@
+Source: neofs-node
+Section: neofs
+Priority: optional
+Maintainer: Dmitriy Zabolotskiy <d.zabolotskiy@yadro.com>
```

Comment:
Could you please change this line to the group email address for opensource support - NSPCC <tech@nspcc.ru> .
Thanks